.env.sample

```
@@ -1,33 +1,37 @@
-MONGODB_URL=mongodb://127.0.0.1:27017
-MONGODB_PORT=27017
+# Mongo DB
+MONGODB_URL=mongodb://127.0.0.1:27017 # not used for deployment with Docker Compose
+MONGODB_PORT=27017 # Used only for deployment with Docker Compose
 MONGODB_DBNAME=ban
 
-REDIS_URL=redis://127.0.0.1:6379
-REDIS_PORT=6379
+# Redis
+REDIS_URL=redis://127.0.0.1:6379 # not used for deployment with Docker Compose
+REDIS_PORT=6379 # Used only for Docker Compose
 
+# APIs
+# API BAN
 BAN_API_URL=https://plateforme.adresse.data.gouv.fr
-BAN_API_AUTHORIZED_TOKENS=
+ADMIN_TOKEN=  used for legacy routes
+BAN_API_AUTHORIZED_TOKENS= # used for new ban-id api routes
+PORT=
```

```suggestion
PORT=5000
```
pas très important mais on l' perdu dans le transport